### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍 👍

**Review by mcdonnnj — APPROVED** [review]
🤖 ⚡

File: src/pca_gophish.yml [hunk 1]
```diff
@@ -21,25 +21,29 @@
 
     - name: Create mount point for GoPhish persistent data volume
       file:
+        mode: 0644
```
Comment (jsf9k) [inline src/pca_gophish.yml:24]: This mode should be `0755` since it is a directory.

File: src/pca_gophish.yml [hunk 2]
```diff
@@ -21,25 +21,29 @@
...
         path: "{{ pca_gophish_composition_path }}/data"
         state: directory
 
     - name: Change GoPhish published port from 3380 to 80
       lineinfile:
+        mode: 0644
```
Comment (jsf9k) [inline src/pca_gophish.yml:30]: I don't think that it is necessary to specify the mode here.  I thought this was required only for tasks that will generate a new file.  Do you remember @mcdonnnj?

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested a change and brought up a minor issue.

File: src/pca_gophish.yml (hunk 2 above)
Comment (mcdonnnj) [reply src/pca_gophish.yml:30]: `lineinfile` is given a pass unless it has the `create` parameter.

Comment (dav3r) [reply src/pca_gophish.yml]: Ah you are right @jsf9k - I just reviewed my local pre-commit errors and there wasn't anything for that task- I just got overzealous when I was adding those modes.  Removed in https://github.com/cisagov/pca-gophish-composition-packer/pull/22/commits/534bdd2914cc8a4a2f27cc0df4d22e9dfce8b90e.

File: src/pca_gophish.yml (hunk 1 above)
Comment (dav3r) [reply src/pca_gophish.yml]: Thanks! Fixed in https://github.com/cisagov/pca-gophish-composition-packer/pull/22/commits/dfa4c51232c2892092808b148c837e0d082cef1d.

**Review by jsf9k — APPROVED** [review]

**Comment by jsf9k** [comment]
It can be so difficult to hit that zealousness sweet spot.

**Pull request merged**